editor/extension.ts

```diff
@@ -93,14 +92,16 @@ pxt.editor.initExtensionsAsync = function (opts: pxt.editor.ExtensionOptions): P
                             }],
                             htmlBody: `
 <p>
-${lf("Please download again to send your code to the EV3 over Bluetooth.")}
+${lf("Download again to send your code to the EV3 over Bluetooth. Make sure to stop your program!")}
```

Comment:
We should be able to tell the program to stop via serial. We should be also able to pipe out `console.log` via the same connection. I can look into that when I'm back.